dinky-web/src/pages/DataStudio/LeftContainer/Project/constants.tsx

```diff
@@ -63,10 +63,14 @@ export const FOLDER_RIGHT_MENU = (disabled = false): MenuItemType[] => [
     label: l('right.menu.rename')
   },
   {
-    key: 'paste',
```

Comment:
why remove paste???

Reply:
> why remove paste???

I thought it was an unimplemented import before, now it's restored.